test/claimZap.js

```diff
@@ -0,0 +1,223 @@
+const Booster = artifacts.require('Booster');
+const VE3DRewardPool = artifacts.require('VE3DRewardPool');
+const VE3DLocker = artifacts.require('VE3DLocker');
+const VeAssetDepositor = artifacts.require('VeAssetDepositor');
+const ClaimZap = artifacts.require('ClaimZap');
+
+let deployer = '0x2093b4281990A568C9D588b8BCE3BFD7a1557Ebd';
+
+//system
+let veToken;
+let ve3Token;
+let veAsset;
+let depositor;
+let exchange;
+let weth;
+let ve3TokenRewardPool;
+let ve3dRewardPool;
+let ve3dLocker;
+let booster;
+let lpToken;
+let feeToken;
+
+let userA;
+
+let starttime;
+let veAssetBalance;
+let lpTokenBalance;
+let zap;
+
+const reverter = new Reverter(web3);
+
+async function logBalances() {
+  await ve3Token.totalSupply().then(a => console.log('ve3Token total supply: ' + web3.utils.fromWei(a, 'ether')));
+  await veAsset.balanceOf(userA).then(a => console.log('userA veAsset: ' + web3.utils.fromWei(a, 'ether')));
+  await ve3Token.balanceOf(userA).then(a => console.log('userA ve3Token: ' + web3.utils.fromWei(a, 'ether')));
+  await veToken.balanceOf(userA).then(a => console.log('userA veToken: ' + web3.utils.fromWei(a, 'ether')));
+  await lpToken.balanceOf(userA).then(a => console.log('userA lpToken: ' + web3.utils.fromWei(a, 'ether')));
+  await ve3TokenRewardPool.balanceOf(userA).then(a => console.log('pool ve3Token: ' + web3.utils.fromWei(a, 'ether')));
+  await ve3dRewardPool.balanceOf(userA).then(a => console.log('pool veToken: ' + web3.utils.fromWei(a, 'ether')));
+  await ve3TokenRewardPool.earned(userA).then(a => console.log('pool ve3Token earned: ' + web3.utils.fromWei(a, 'ether')));
+  // await ve3dRewardPool.earned(veToken.address, userA).then(a => console.log('pool veToken earned: ' + web3.utils.fromWei(a, "ether")));
+  await ve3dLocker.lockedBalanceOf(userA).then(a => console.log('locked balance: ' + web3.utils.fromWei(a, 'ether')));
+  await ve3dLocker.claimableRewards(userA).then(a => console.log('locked claimableRewards: ' + a));
+}
+
+contract('Test claim zap', async accounts => {
+  before('setup', async () => {
+    veToken = await IERC20.at(contractList.system.vetoken);
+    ve3Token = await IERC20.at(contractList.system.ve3_idle);
+    veAsset = await IERC20.at(contractList.system.idle_address);
+    depositor = await VeAssetDepositor.at(contractList.system.idle_depositor);
+    exchange = await IExchange.at('0xd9e1cE17f2641f24aE83637ab66a2cca9C378B9F');
+    weth = await IERC20.at('0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2');
+    ve3TokenRewardPool = await BaseRewardPool.at(contractList.system.idle_ve3TokenRewardPool);
+    ve3dRewardPool = await VE3DRewardPool.at(contractList.system.vetokenRewards);
+    ve3dLocker = await VE3DLocker.at(contractList.system.ve3dLocker);
+    booster = await Booster.at(contractList.system.idle_booster);
+    lpToken = await IERC20.at(contractList.system.idle_lptoken);
+    feeToken = await IERC20.at(await booster.feeToken());
```

Comment:
can you adjust this test file to work against the current network weather idle or angle like other test files